doc/snippets/Microsoft.Data.SqlClient.DataClassification/SensitivityClassification.xml

```diff
@@ -3,26 +3,47 @@
     <members name="SensitivityClassification">
         <SensitivityClassification>
             <summary>Provides the functionlity to retrieve Sensitivity Classification data as received from SQL Server for the active 'SqlDataReader'</summary>
-            <remarks>To be added.</remarks>
         </SensitivityClassification>
         <ColumnSensitivities>
             <summary>Returns the column sensitivity for this 'SensitivityClassification' Object</summary>
             <value>List of column sensitivities.</value>
-            <remarks>To be added.</remarks>
         </ColumnSensitivities>
         <InformationTypes>
             <summary>Returns the information types collection for this 'SensitivityClassification' Object</summary>
             <value>List of information types.</value>
-            <remarks>To be added.</remarks>
         </InformationTypes>
         <Labels>
             <summary>Returns the labels collection for this 'SensitivityClassification' Object</summary>
             <value>List of labels.</value>
-            <remarks>To be added.</remarks>
         </Labels>
+        <SensitivityRank>
+          <summary>Returns the relative sensitivity rank for the query associated with the active 'SqlDataReader'.</summary>
+          <value>Relative sensitivity ranking for this query.</value>
+          <remarks>
+            <format type="text/markdown">
+              <![CDATA[
+
+## Remarks
+A relative ranking of the sensitivity of a query or of a column that is part of percolumn data. Available values are as below:
```

Comment:
Since this is the rank for the reader, it should only mention the query? Same thing with column in SensitivityProperty.
```suggestion
A relative ranking of the sensitivity of a query. Available values are as below:
```